### PR TITLE
get_valid_regions()

### PR DESCRIPTION
**Pull request by hanpeter**
## What does this PR do?
This PR adds `get_valid_regions()` to `krux_boto.boto.Boto` classes as well as augment `krux_boto.util.RegionCode` with all current AWS regions.

## Why is this change being made?
To future proof `krux_boto.util.RegionCode`.

## Where should the reviewer start?
`krux_boto.boto.Boto.get_valid_regions()`

## How was this tested? How can the reviewer verify your testing?
The change was manually tested on the development environment and through the unit test.

## Completion checklist

- [x] The pull request has been appropriately labelled according to
  our [conventions](https://kruxdigital.jira.com/wiki/display/EN/Changes+and+Peer+Review)
- [x] The change has unit & integration tests as appropriate.
- [x] Documentation has been updated.

**NOTE:** This PR depends on #25 and is currently against #25. Once #25 is merged, this PR will be modified to compare against the master.

### CODE REVIEW

File: krux_boto/util.py [hunk 1]
```diff
@@ -47,19 +47,41 @@ def get_instance_region():
...
+        PDX = 2   # Portland, Oregon
+        DUB = 3   # Dublin, Ireland
+        SIN = 4   # Singapore
+        BOM = 5   # Mumbai (Bombay), India
+        SYN = 6   # Sydney, Australia
+        FRA = 7   # Frankfurt, Germany
+        NRT = 8   # Tokyo (Narita), Japan
+        ICN = 9   # Seoul (Incheon), South Korea
+        GRU = 10  # Sao Paulo (Guarulhos), Brazil
+        SFO = 11  # San Francisco, California
```
Comment (paul-krohn) [inline krux_boto/util.py]: isn't us-west-1 in San Jose?

Comment (hanpeter) [inline krux_boto/util.py]: Is it? They only say North California.

Comment (paul-krohn) [inline krux_boto/util.py]: They don't say, but they also don't even have a CloudFront edge location in SF, but do in San Jose; if you geo-locate an IP in us-west-1, it says San Jose.

**Comment by paul-krohn** [comment]
👍

**Pull request merged**